bin/write-good.js

```diff
@@ -20,16 +20,34 @@ if (files.length === 0) {
   process.exit(1);
 }
 
-var opts      = {
-  weasel   : null,
-  illusion : null,
-  so       : null,
-  thereIs  : null,
-  passive  : null,
-  adverb   : null,
-  tooWordy : null,
-  cliches  : null
-};
+//set custom ops, i.e. to lint a non-English document
+var optsArg = args.find(function (arg) {
```

Comment:
Why use the optsArg variable at all? Just use checksModule, and put this functionality in line 28. A bit cleaner.